azure/azure_test.go

```diff
@@ -82,7 +92,7 @@ func Test_Examples(t *testing.T) {
 		{
 			Dir:     path.Join(cwd, "./examples/table"),
```

Comment:
I assume we can also remove some of the duplicate tests we added specifically for Azure and rely on shared tests?

Reply:
Correct.

Reply:
This is all just happening in small steps though so i don't have a massive PR that i'm trying to juggle.